src/YagrCore/plugins/plotLines/plotLines.ts

```diff
@@ -130,21 +98,14 @@ export default function plotLinesPlugin(options: PlotLineOptions): PlotLinesPlug
                         }
                     }
                 } else {
-                    const from = u.valToPos(value, scale, true);
                     const pConf = plotLineConfig as PLineConfig;
-
-                    ctx.beginPath();
-
                     if (scale === DEFAULT_X_SCALE) {
-                        /** Workaround to ensure that plot line will not be drawn over axes */
```

Comment:
can we keep this comment? Is it still actual or not?

Reply:
Why do we need this changes? two row code is more readable then single

Reply:
I will keep comment and return some changes (moved the calculation for from/to to the upper context, but forgot ctx.beginPath();